packages/protocol/scripts/foundry/constants.sh

```diff
@@ -32,6 +32,8 @@ export ANVIL_FOLDER="$TMP_FOLDER/devchain"
 # Contract addresses
 export REGISTRY_ADDRESS="0x000000000000000000000000000000000000ce10"
 export PROXY_ADMIN_ADDRESS='0x4200000000000000000000000000000000000018' # This address is defined in `IsL2Check.sol`
+export ADDRESS_ZERO="0x0000000000000000000000000000000000000000"
+export CELO_VM_ADDRESS=$ADDRESS_ZERO
```

Comment:
Nit: to be consistent with variable names above

```suggestion
export ZERO_ADDRESS="0x0000000000000000000000000000000000000000"
export CELO_VM_ADDRESS=$ZERO_ADDRESS
```